components/concept/sync/src/main/java/mozilla/components/concept/sync/OAuthAccount.kt

```diff
@@ -56,24 +56,24 @@ interface AccountObserver {
     /**
      * Account just got logged out.
      */
-    fun onLoggedOut()
+    fun onLoggedOut() = Unit
 
     /**
      * Account was successfully authenticated.
      * @param account An authenticated instance of a [OAuthAccount].
      */
-    fun onAuthenticated(account: OAuthAccount)
+    fun onAuthenticated(account: OAuthAccount, newAccount: Boolean = false) = Unit
```

Comment:
This is a breaking change for consumers that implement AccountObserver, right? Let's document it.

Reply:
With a default param, this shoudn't be a breaking change, but I forgot to add it to the changelog!

Reply:
Ignore, I saw your comment below.